src/pages/Timeline.tsx

```diff
@@ -18,7 +18,7 @@ const Timeline = () => {
           <Typography sx={styles.headerTitle} variant="h2">
             Timeline
           </Typography>
-          <ComposePost placeholder="What's happening?" />
+          <ComposePost placeholder="What's happening?" onClose={() => {}} />
```

Comment:
If we make `onClose` an optional prop, we can delete this
```suggestion
          <ComposePost placeholder="What's happening?"/>
```

Reply:
Alright Cap, done